### ISSUE TITLE
KeyError: 'subaccountid' causing security context create to fail

### ISSUE THREAD

**Issue by mattreddy3** [SAP/cloud-pysec#45]
https://github.com/SAP/cloud-pysec/blob/107ce168e3ce20d6a69cf4b6588bb4199c1a2bef/sap/xssec/security_context.py#L274

Hello,

I have just successfully set up sap_xssec in my cloud foundry python application, and I am getting this error:

```2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR Traceback (most recent call last):
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR   File "/home/vcap/deps/0/python/lib/python3.8/site-packages/flask/app.py", line 1982, in wsgi_app
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR     response = self.full_dispatch_request()
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR   File "/home/vcap/deps/0/python/lib/python3.8/site-packages/flask/app.py", line 1614, in full_dispatch_request
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR     rv = self.handle_user_exception(e)
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR   File "/home/vcap/deps/0/python/lib/python3.8/site-packages/flask/app.py", line 1517, in handle_user_exception
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR     reraise(exc_type, exc_value, tb)
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR   File "/home/vcap/deps/0/python/lib/python3.8/site-packages/flask/_compat.py", line 33, in reraise
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR     raise value
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR   File "/home/vcap/deps/0/python/lib/python3.8/site-packages/flask/app.py", line 1612, in full_dispatch_request
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR     rv = self.dispatch_request()
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR   File "/home/vcap/deps/0/python/lib/python3.8/site-packages/flask/app.py", line 1598, in dispatch_request
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR     return self.view_functions[rule.endpoint](**req.view_args)
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR   File "hello.py", line 24, in hello
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR     security_context = xssec.create_security_context(access_token, uaa_service)
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR   File "/home/vcap/deps/0/python/lib/python3.8/site-packages/sap/xssec/__init__.py", line 13, in create_security_context
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR     return SecurityContext(token, uaa_service)
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR   File "/home/vcap/deps/0/python/lib/python3.8/site-packages/sap/xssec/security_context.py", line 45, in __init__
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR     self._init_properties()
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR   File "/home/vcap/deps/0/python/lib/python3.8/site-packages/sap/xssec/security_context.py", line 51, in _init_properties
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR     self._offline_validation()
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR   File "/home/vcap/deps/0/python/lib/python3.8/site-packages/sap/xssec/security_context.py", line 315, in _offline_validation
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR     self._set_ext_attr(jwt_payload)
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR   File "/home/vcap/deps/0/python/lib/python3.8/site-packages/sap/xssec/security_context.py", line 274, in _set_ext_attr
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR     self._logger.debug('Obtained subaccountid: %s.', self._properties['subaccountid'])
   2020-09-25T12:43:12.73+0200 [APP/PROC/WEB/0] ERR KeyError: 'subaccountid'
```

In checking out the source code, this line seems to be the culprit:
```
if 'subaccountid' in ext_attr and ext_attr.get('subaccountid'):
                self._properties['subaccount_id'] = ext_attr.get('subaccountid')
                self._logger.debug('Obtained subaccountid: %s.', self._properties['subaccountid'])
```
This seems a pretty simple change- just fix the property reference in the debug logger to `'subaccount_id'`, no? I am not an experienced python programmer so perhaps I am mistaken.

Thanks!

**Comment by robertofalk** [comment]
Fixed by #46

**Comment by robertofalk** [comment]
Thanks!
